core/controller/src/main/scala/whisk/core/controller/Authenticate.scala

```diff
@@ -59,25 +59,25 @@ trait Authenticate {
       Try {
         // authkey deserialization is wrapped in a try to guard against malformed values
         val authkey = AuthKey(UUID(pw.username), Secret(pw.password))
-        logging.info(this, s"authenticate: ${authkey.uuid}")
+        logging.debug(this, s"authenticate: ${authkey.uuid}")
```

Comment:
this is useful to be able to correlate a tid to a uuid. i would revert this one.